solver/output.go

```diff
@@ -0,0 +1,47 @@
+package solver
+
+import (
+	"bufio"
+	"log"
+	"os"
+)
+
+const (
+	OutputEndOfLine = "\n"
+	OutputDelimiter = " "
+)
+
+type Serializable interface {
+	String() string
+}
+
+func DumpToFile(filename string, output *[][]Serializable) {
```

Comment:
C'est plus simple de prendre les mêmes paramètres que pour l'input non ? Par la j'entends une `map[int][]string`
ou la clef représente la ligne et la value les valeurs séparées par des ' '

Reply:
ok, specific method for that was created in the new commit